.circleci/config.yml

```diff
@@ -367,19 +453,20 @@ workflows:
   version: 2
 
   build-and-deploy-all-apps-workflow:
+    unless: << pipeline.parameters.do-builds >>
     jobs:
-      - osteo-build-and-deploy: &build-and-deploy-filters
+      - conditionally-launch-build-and-deploy-all-job: &build-and-deploy-filters
           filters:
             branches:
               only:
                 - develop
-                - angular-appengine-deployment-ctd-2
-      - brain-build-and-deploy:
-          <<: *build-and-deploy-filters
-      - angio-build-and-deploy:
-          <<: *build-and-deploy-filters
-      - mbc-build-and-deploy:
-          <<: *build-and-deploy-filters
+                - circle-ci-updates-1
+                - /play.*/
+
+  build-and-deploy-workflow:
+    when: << pipeline.parameters.do-builds >>
```

Comment:
This << pipeline.parameters.do-builds >> param is used by both these workflows. Since they both trigger on changes to the branches, this is the trick to allow one workflow to call the other. The boolean do-builds get set to true when the build is initiated by an API call in the command `conditionally-launch-build-and-deploy`